streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBSessionStore.java

```diff
@@ -85,11 +86,21 @@ public void init(final ProcessorContext context, final StateStore root) {
 
     @Override
     public void remove(final Windowed<K> key) {
-        bytesStore.remove(Bytes.wrap(SessionKeySchema.toBinary(key, serdes.keySerializer(), topic)));
+        try {
+            bytesStore.remove(Bytes.wrap(SessionKeySchema.toBinary(key, serdes.keySerializer(), topic)));
+        }  catch (final ProcessorStateException e) {
```

Comment:
Do we need this change here? `RocksDBSessionStore` should be wrapped with `MeteredSessionStore`. As an alternative, we can remove the code from `MeteredSessionStore` thought. Not sure, which class is better for the fix.

\cc @guozhangwang WDYT?

Reply:
Done